guides/include-cleaner.md

```diff
@@ -76,19 +105,42 @@ section). You may have to add missing includes too, or do other cleanups.
 - Identify which headers are **directly** used, and `#include` those instead.
 - Suppress the warning with `// IWYU pragma: keep`.
 
+**clangd suggests `#include "X"`, but the real header for the symbol is Y.**
+
+This is merely a suggestion: including `Y` should also hide the warning.
+
+clangd tries to pick the best header, preferring those that are public,
+self-contained, which provide full definitions (for classes and templates),
```

Comment:
remove "which".

Reply:
Ugh, this sentence is a mess, sorry.
I don't think just removing "which" is enough, had to rephrase the last list item too.